src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -295,6 +398,15 @@ private static bool IsValueTypeErrorOrBlank<T>(FormulaValue val)
             return new InMemoryTableValue(irContext, pairs.Select(pair => pair.Key));
         }
 
+        private static int CompareColumns<TPFxPrimitive, TDotNetPrimitive>(FormulaValue a, FormulaValue b)
```

Comment:
>CompareColumns [](http://example.com/codeflow?start=27&length=14)

This is elegant - but semantically is it consistent with the existing binary comparer?

See OperatorBinaryLeq etc in LibraryOperators.

This could be an issue for blanks, errors. and possibly string casing.